azure-pipelines-nightly.yml

```
@@ -639,6 +639,24 @@ jobs:
           PackageName: ESP32_P4_USB
           CMakePreset: ESP32_P4_USB
 
+        ESP32_P4_USB:
```

YAMLlint (1.38.0)

[error] duplication of key "ESP32_P4_USB" in mapping

(key-duplicates)